local-test/per-panel-audit.py

```
@@ -0,0 +1,189 @@
+            all_layout_issues[row] = layout_issues
+            out.append("\n**Layout issues**:")
+            for issue in layout_issues:
+                out.append(f"- ⚠️ {issue}")
+        else:
+            out.append("\nLayout: ✅ all rows fill 24 columns cleanly")
+
+        out.append("\n| ID | Title | Type | Pos | Size | Status | Note |")
+        out.append("|---:|---|---|---|---|---|---|")
+        for p in panels:
+            g = p["gridPos"]
+            status, n, note = classify_panel(p)
```

Ruff (0.15.10)

[warning] Unpacked variable `n` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)